src/engraving/dom/engravingitem.cpp

```diff
@@ -1462,6 +1462,13 @@ PropertyPropagation EngravingItem::propertyPropagation(const EngravingItem* dest
     const Score* sourceScore = score();
     const Score* destinationScore = destinationItem->score();
     const bool isTextProperty = propertyGroup(propertyId) == PropertyGroup::TEXT;
+    const Staff* sourceStaff = staff();
+    const Staff* destinationStaff = destinationItem->staff();
+
+    // Properties must be propagated to items cloned for MMRests
+    if (sourceScore == destinationScore && sourceStaff == destinationStaff) {
```

Comment:
Currently, properties aren't propagated to items in the same score.  I think they definitely should be for clones inside/outside MMRests - see below.

https://github.com/user-attachments/assets/751db5fd-c5ab-494d-a2ac-a0b8d680c253

It doesn't seem the most robust way of detecting this specific situation, but I don't think we ever have clones on the same score & staff apart from with MMRests.